File: 0075-sort-colors/0075-sort-colors.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) 
    {
        int numOfZeros = 0,numOfOnes = 0,numOfTwos = 0;
            
        for(int i = 0 ; i< nums.size(); i++)
        {
            numOfZeros+=(nums[i]==0);
            numOfOnes+=(nums[i]==1);
            numOfTwos+=(nums[i]==2);
        }
        int index = 0;
        while(numOfZeros > 0){nums[index] = 0;index++;numOfZeros--;}
        while(numOfOnes > 0){nums[index] = 1;index++;numOfOnes--;}
        while(numOfTwos > 0){nums[index] = 2;index++;numOfTwos--;}
    }
};

File: 0075-sort-colors/0075-sort-colors.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        freq = [0]*3
        for i in nums:freq[i] += 1;
        
        for i in range(freq[0]):nums[i] = 0
        for i in range(freq[1]):nums[i+freq[0]] = 1
        for i in range(freq[2]):nums[i+freq[0]+freq[1]] = 2
            
            
        return nums